bin/pqcheck

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bin/ruby
+
+DEFAULT_PQ_DIR = "data/queue/main"
+
+argv0 = ARGV[0].to_s.strip
+if argv0 == "-h" || argv0 == "--help"
+  puts("usage: pqcheck [PQ dir path]\n  default [PQ dir path] is #{DEFAULT_PQ_DIR}")
+  exit(0)
+end
+
+dir = argv0.empty? ? DEFAULT_PQ_DIR : argv0
+if (!Dir.exist?(dir))
+  puts("error: invalid PQ dir path: #{dir}")
+  exit(1)
+end
+
+puts("checking queue dir: #{dir}")
+
+Dir.glob("#{dir}/checkpoint.*").sort_by { |x| x[/[0-9]+$/].to_i }.each do |checkpoint|
+  data = File.read(checkpoint)
```

Comment:
If valid checkpoints are all in theory 33 bytes, should we just read the first 33 bytes (or 33+1 bytes)?